WalletWasabi.Fluent/App.axaml.cs

```diff
@@ -84,6 +84,7 @@ private void DesktopOnShutdownRequested(object? sender, ShutdownRequestedEventAr
 		{
 			if (CanShutdownProvider is { } provider)
 			{
+				// Shutdown prevention will only work if you directly run the executable.
```

Comment:
This is strage, yesterday it worked well with VS and `dotnet run` too. I need to look into this.